### DISCUSSION TITLE
Interaction Design - Further Reading / Training

### DISCUSSION THREAD

**Discussion by nhsbsa-sthil** [nhsbsa/nhsbsa-digital-playbook#312] [UCD]
I've begun to put together a list of further reading / training courses from a B6 & B7 Interaction Design point of view:

https://miro.com/app/board/uXjVN1D-srg=/?share_link_id=180528106724

These are broken down based on the skills required within the Government Digital and Data Profession Capability Framework:

https://ddat-capability-framework.service.gov.uk/interaction-designer.html

I'm in the process of getting feedback from other Senior Interaction Designers, and I hope that once done, we can add it into the playbook.

**Comment by nhsbsa-sthil** [comment]
Added a page into the playbook under the DRAFT status. Any comments / changes recommended!

https://nhsbsa.github.io/nhsbsa-digital-playbook/design/interaction-designer/training-resources/

**Reply by nhsbsa-thole** [reply]
With the titles for band 6 and band 7 is there ever a case where a band 7 is not called a 'Senior Interaction Designer'? If not it might be better to reference the difference via role than band number.

**Reply by nhsbsa-sthil** [reply]
To be honest - mentioning the bands is probably a bit superfluous, so they should probably be removed. 👍 

In terms of the difference between the 2 roles - this tends to come from the level of experience within each skill area e.g.

You'll notice that a Interaction Designer has a 'working' level of skill for Agile Working, whereas a Senior Interaction Designer has a 'practitioner' level of skill.

**Reply by nhsbsa-thole** [reply]
Oh, I just meant that the band 7 H2 title doesn't say "Senior" so was wondering if we should?

**Reply by nhsbsa-sthil** [reply]
Ah! That's my bad! Definitely an oversight! It should say 'Senior Interaction Designer' - I'll get that fixed 👍

**Comment by nhsbsa-sthil** [comment]
Closed and live:

https://nhsbsa.github.io/nhsbsa-digital-playbook/design/interaction-designer/training-resources/
